test/external/imports/named-import/ts-cjs/source/index.ts

```diff
@@ -1,7 +1,7 @@
 // /source/index.ts
 // Run the server
 
-import app from './app'
+import { app } from './app'
```

Comment:
how did that work before...?

Reply:
I was shocked too, I have literally no idea how it worked before